scripts/Config-Dev.js

```diff
@@ -4,6 +4,7 @@ Config = {
 	RPC_URL: 'https://testapi.steem-engine.com/',
 	ACCOUNTS_API_URL: 'https://testaccounts.steem-engine.com',
 	CONVERTER_API: 'https://converter-api.steem-engine.com/api',
+	SCOT_API: 'http://scot-api.steem-engine.com/',
```

Comment:
The https address works also:
SCOT_API: 'https://scot-api.steem-engine.com/',

Reply:
Good catch. This should be https